libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md

```diff
@@ -6,7 +6,8 @@ The SPRacingH7 Extreme is a flight controller produced by [Seriously Pro Racing]
 
  - MCU - STM32H750 32-bit processor running at 400 MHz
  - 16MByte Serial NOR flash via QuadSPI
- - IMUs - 2x ICM20602
+ - 16MB Dataflash
+ - IMUs - 2x ICM20602 (only one used due to CPU limitations)
```

Comment:
Only one enabled by default - it's perfectly possible for the user to enable both if they want

Reply:
added